dracut-module-setup.sh

```diff
@@ -666,8 +666,6 @@ kdump_install_net() {
     local _static _proto _ip_conf _ip_opts _ifname_opts
     local _znet_netdev _znet_conpath
 
-    clone_nmconnections
-
```

Comment:
why is this line?

Reply:
Sorry for the confusion! If you check the tab [Files changed](https://github.com/coiby/kexec-tools/pull/9/files), this line isn't there. Maybe the issue is similar to `can_ipv4_or_ipv6_fail`.

Reply:
Oops, it turns outs this was to fix a mistake introduced in previous commit. Now fixed in new version. So is `can_ipv4_or_ipv6_fail`. Thanks for catching it!